test/distributed/test_c10d_nccl.py

```diff
@@ -3924,6 +3924,8 @@ class NCCLTraceTestDumpOnTimeout(NCCLTraceTestDumpOnTimeoutBase):
     @skip_but_pass_in_sandcastle_if(not TEST_MULTIGPU, "NCCL test requires 2+ GPUs")
     @parametrize("timing_enabled", [True, False])
     def test_timeout_dumps(self, timing_enabled):
+        # We need to reduce the coordinated check frequency to avoid rank 0 also timeout.
+        os.environ['TORCH_NCCL_COORD_CHECK_MILSEC'] = '15000'
```

Comment:
ok. Given that these tests are generally pretty slow (many tests take 10-20 second to run), choosing a number like '15s' for "a long enough time to not happen" looks risky - thats what originally motivated me to ask this question.

IF you are just trying to disable coordination, then i'd suggest setting a REALLY large number and update the comment to be explicit that you're trying to avoid coordinated dump so that only rank0 reports timeout

Reply:
Sure. Let me do that.